app/src/main/res/xml/sensor_preferences.xml

```diff
@@ -0,0 +1,47 @@
+<PreferenceScreen xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:andriod="http://schemas.android.com/apk/res/android">
+
+    <PreferenceCategory
+        andriod:key="sensor_category"
+        andriod:title="Sensor settings">
+
+        <EditTextPreference
+            andriod:key="ip"
+            andriod:title="IP address"
+            app:useSimpleSummaryProvider="true" />
+
+        <EditTextPreference
+            andriod:key="name"
+            andriod:title="Name"
+            app:useSimpleSummaryProvider="true" />
+
+        <ListPreference
+            andriod:entries="@array/icons"
+            andriod:entryValues="@array/iconsValues"
+            andriod:key="changeIcon"
+            andriod:title="Icon"
+            app:useSimpleSummaryProvider="true" />
+
+        <ListPreference
+            andriod:entries="@array/colours"
+            andriod:entryValues="@array/coloursVaules"
+            andriod:key="changeColour"
+            andriod:title="change colour of icon"
+            app:useSimpleSummaryProvider="true" />
```

Comment:
coloursVaules is a typo, Android studio should point out things like that.
The title text should be as short as possible and start with an upper case letter. "Icon colour"

Edit: This doesn't matter since you should remove the colours for now.